app/javascript/packages/document-capture/components/unknown-error.tsx

```diff
@@ -27,12 +43,21 @@ function UnknownError({
     article: 'how-to-add-images-of-your-state-issued-id',
     location: 'document_capture_review_issues',
   });
+
+  const acceptedIdUrl = getHelpCenterURL({
+    category: 'verify-your-identity',
+    article: 'accepted-identification-documents',
+    location: 'document_capture_review_issues',
+  });
+
   const errs =
     !!unknownFieldErrors &&
     unknownFieldErrors.filter((error) => !['front', 'back'].includes(error.field!));
   const err = errs.length !== 0 ? errs[0].error : null;
   if (isFailedDocType && !!altFailedDocTypeMsg) {
-    return <p key={altFailedDocTypeMsg}>{altFailedDocTypeMsg}</p>;
+    return (
+      <p key={altFailedDocTypeMsg}>{formattedIdTypeMsg({ altFailedDocTypeMsg, acceptedIdUrl })}</p>
```

Comment:
Do you still need the <p> tag here? Maybe for formatting reasons?

Reply:
@charleyf , not sure, prefer to leave it there just in case.